code/modules/mob/living/silicon/ai/freelook/eye.dm

```diff
@@ -130,7 +130,9 @@
 // This is handled in the proc below this one.
 
 /client/proc/AIMove(n, direct, mob/living/silicon/ai/user)
-
+	if(world.time < move_delay)
+		return
+	move_delay = world.time + world.tick_lag // cameras and AI eyes still move tile by tile so we need a delay here

```

Comment:
why tick_lag instead of some more concrete delay like `0.5 SECONDS`?

Reply:
so you can't move multiple times per tick, which is probably what was causing the issue. This is what we had before in the base move code

Reply:
What we had before was greater or less delay of multiple ticks, also how would they be moving multiple times per tick when movement is managed by a subystem?